src/markdown-pages/docs/managers/vtp-manager.md

```diff
@@ -0,0 +1,78 @@
+---
+path: "/docs/managers/vtp-manager"
+title: "VTP Manager"
+---
+
+**Release**
+  
+## Overview
+  
+Create an automated integration test in Galasa and use the VTP Manager to transparently record the interactions between the test and your CICS programs. Play back the recorded file by using IBM Z Virtual Test Platform (ZVTP). Playing back the recording in ZVTP means that you can test a change to your application code without running the test inside CICS. Run the recorded tests as part of your regression testing strategy before and after code changes to detect any anomalies. 
+
+The VTP Manager can configure ZVTP by using CICS transactions to start and stop recording on a CICS region. After all recordings are complete, the VTP recording is exported as a sequence of flat files to the specified HLQ.  
+
+Recordings are saved in the format _<HLQ_FROM_CPS>.RUNID.R<number>_, where _number_ is the number of the recorded test method. For example, _CTS.JBLOGGS.VTP.R1234.R1_.
```

Comment:
can we make this format a bit more consistent.  e.g the run ID is also a parameter